tests/components/risco/test_config_flow.py

```diff
@@ -277,6 +277,40 @@ async def test_local_form(hass: HomeAssistant) -> None:
     mock_close.assert_awaited_once()
 
 
+async def test_local_form_with_one_error(hass: HomeAssistant) -> None:
```

Comment:
I don't think we should automatically set this on error. So far, it seems like your system is the only one that needs a communication delay, and there's no reason to add it in case of some intermittent error. Let's keep it explicit.

Reply:
I concluded from comments https://github.com/home-assistant/core/issues/130660#issuecomment-2486570639 and https://github.com/OnFreund/pyrisco/issues/27#issuecomment-2553502232 that you wanted to include the use of the communication delay again in the initialization.
This test is just an additional test to the already existing code.
Back when this functionality was initially added to the integration, I added the communication delay as a configuration setting but this was considered to be not user friendly (https://github.com/home-assistant/core/pull/101349#discussion_r1366111030). That's why this automatic determination of the communication delay was introduced. Are you suggesting to go back to a configuration parameter for this?

Reply:
Yeah, I remember that conversation, but we've learned two things since:
1. This seems to only affect one person
2. Intermittent network errors are more common (users opening bug reports had random communication delays, I saw my system's communication delay increasing every now and then).

Given those, I think explicit is the way to go.

Reply:
03397 also seems to be affected by this but I get your point that this is a rather exceptional case.

The change in this PR, apart from the added test, will just use the communication delay that has been determined when the integration was set up. It won't go up afterwards since that code is not added again.
One thing I'm wondering: if a user wants to setup the integration and he would also have a slower responding panel, he will most likely need to try several times to setup the integration if we remove the determination of the delay. In my case, without the communication delay, setting up the integration fails 9 out of 10 times. How would you like to cover that because it can also generate bug reports related to this issue.

Reply:
We can potentially ask for the communication delay when setting the integration up. We can limit it just to advanced mode.